OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -194,13 +194,13 @@ private void Pane_DblClickHandler(object sender, EventArgs e)
         {
             try
             {
-                ExpandedImageForm expandedImageForm;
+                ExpandedImageDialog expandedImageForm;
```

Comment:
MainForm should have just taken the place of ExpandedImageForm.  In the future remove the class to be replaced  and rename the old one. Also be mindful when renaming classes to adhere to current standards and practices. 

Changing the name of a class especially in another class to something completely different is a big no-no in a collaborative environment because you will cause conflicts if someone else is working on TiledFilterForm.